topics/docker.md

```diff
@@ -176,13 +197,14 @@ ENTRYPOINT ["java","-jar","/app/ktor-docker-sample.jar"]
 </tab>
 <tab title="Maven" group-key="maven">
 
+
 ```Docker
 FROM maven:3-openjdk-11 AS build
```

Comment:
The caching step is missing in the Maven version.

Reply:
Yes, I did not touch Maven because I dont know what to write, Maven docker image dropped support for openjdk after 18.

```
<warning>
OpenJDK version in build stage is 18, and official Maven image dropped support for
OpenJDK after version 18. Please refer to the information below for replacements
to OpenJDK.
</warning>

```Docker
FROM gradle:latest AS cache
RUN mkdir -p /home/gradle/cache_home
ENV GRADLE_USER_HOME /home/gradle/cache_home
COPY build.gradle.* gradle.properties /home/gradle/java-code/
WORKDIR /home/gradle/java-code
RUN gradle clean build -i --stacktrace

FROM maven:3-openjdk-18 AS build
COPY . /home/maven/src
WORKDIR /home/maven/src
RUN mvn package

FROM openjdk:22
EXPOSE 8080:8080
RUN mkdir /app
COPY --from=build /home/maven/src/target/*-with-dependencies.jar /app/ktor-docker-sample.jar
ENTRYPOINT ["java","-jar","/app/ktor-docker-sample.jar"]
```
something like this would be ok?